### PR TITLE
Update README, Examples (v1beta3) & Usage

### PR DESCRIPTION
**Pull request by karlkfi**
Update examples to use v1beta3 and add an intro doc for using them.

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+###Intro to Kubernetes on Mesos
```
Comment (tsenart) [inline docs/intro.md]: It goes easier on the eye if some spacing is added before the text (IMO). `### Intro (...)`

Comment (karlkfi) [inline docs/intro.md]: Last time I tried that it didn't show up as a header on github...

Comment (tsenart) [inline docs/intro.md]: When was that? :-)

File: examples/service-nginx.json [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+  "metadata": {
+    "name": "nginx"
+  },
+  "spec": {
+    "selector": {
+      "name": "nginx"
+    },
+    "ports": [
+      {
+        "protocol": "TCP",
```
Comment (jdef) [inline examples/service-nginx.json]: define a service port name here, will allow this to be used with a replicated nginx pod

File: docs/intro.md (hunk 1 above)
Comment (karlkfi) [inline docs/intro.md]: I stand corrected.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,43 @@
 kubernetes-mesos
 ================
 
-When [Google Kubernetes][2] meets [Apache Mesos][3]
+[Google Kubernetes](https://github.com/GoogleCloudPlatform/kubernetes) on [Apache Mesos](http://mesos.apache.org/)
 
-Kubernetes and Mesos are a match made in heaven.
-Kubernetes enables the Pod, an abstraction that represents a group of co-located containers, along with Labels for service discovery, load-balancing, and replication control.
-Mesos provides the fine-grained resource allocations for pods across nodes in a cluster, and facilitates resource sharing among Kubernetes and other frameworks running on the same cluster.
+Disclaimer: Kubernetes is pre-release quality, and not yet recommended for production systems.
```
Comment (jdef) [inline README.md]: can we say "Kubernetes-Mesos" here instead? Let the k8s team make their own announcements w/ respect to production fitness

File: docs/usage.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+<a name="service"></a>
+#### Create a Kubernetes Service Definition
+
+A [service inside kubernetes](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md) is an abstraction that gives a pod (or an external service) a named endpoint.
+In order to expose the nginx server to the web, you must create a Kubernetes service that refers to it.
+
+1. Download an example nginx service definition:
+
+    ```
+    mkdir -p examples && curl -o examples/service-nginx.json https://raw.githubusercontent.com/mesosphere/kubernetes-mesos/master/examples/service-nginx.json
```
Comment (jdef) [inline docs/usage.md]: we're asking the user to download the v0.5.0 binaries, but not the v0.5.0 examples?

Comment (karlkfi) [inline docs/usage.md]: dunno. we kind of have two audiences: those using master head and those using a specific version... We don't have dev binaries being built and hosted automatically yet. So we can't just tell them to download everything. Maybe we should just have instructions for master...

Comment (sttts) [inline docs/usage.md]: Hopefully also the API will stabilize with the 1.0. Then those examples will not change that often anymore.

File: README.md [hunk 5]
```diff
@@ -1,27 +1,43 @@
...
+The [getting started guide](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/mesos.md)
+is in the Kubernetes repo.
 
-[1]: DEVELOP.md
-[2]: https://github.com/GoogleCloudPlatform/kubernetes
-[3]: http://mesos.apache.org/
-[4]: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/mesos.md
-[5]: docs/README.md
-[6]: docs/issues.md
+The [introductory usage guide](docs/usgae.md) is in the docs dir.
```
Comment (sttts) [inline README.md]: Typo: usage.md

File: README.md [hunk 6]
```diff
@@ -1,27 +1,43 @@
...
 
-Refer to the [documentation pages][5] for a review of core concepts, system architecture, and [known issues][6].
+----------------
 
-The project has been migrated upstream into the offical [Kubernetes repository][2].
-Please see the [build documentation][1] if you are interested in building from source.
+Kubernetes is an open source container-orchestration project introduced by Google.
+
+Kubernetes itself is deployable to almost any cluster of (virtual or real) machines, in your own datacenter or on third
+party infrastructures. With the Mesos integration layer, Kubernetes acts as a Mesos framework, scheduling Pods as Mesos
```
Comment (sttts) [inline README.md]: Pods => pods

Comment (sttts) [inline README.md]: Pods/pods is not consistent throughout the whole file.

File: docs/usage.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+- [Create a Kubernetes Pod Definition](#pod)
+- [Create a Kubernetes Service Definition](#service)
+- [Launch a Kubernetes Pod via REST API](#api)
+- [Launch a Kubernetes Pod via kubectl](#kubectl)
+- [View Usage Metrics](#metrics)
+
+<a name="config"></a>
+#### Configure the Kubernetes Endpoint
+
+Export the Kubernetes URL to an environment variable, where `<hostname>` is the Mesos Master hostname:
```
Comment (sttts) [inline docs/usage.md]: Better "to the 'KUBERNETES_MASTER' env var" because for kubectl the name is important.

**Comment by sttts** [comment]
A few nits. Otherwise, LGTM

File: docs/usage.md [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+
+    - If you're using a released version of Kubernetes:
+        Download the kubectl binary [specific to that release](https://github.com/mesosphere/kubernetes-mesos/releases),
+        appropriate for your OS and architecture. For example, v0.5.0 release binaries can be found on the
+        [v0.5.0 release tag](https://github.com/mesosphere/kubernetes-mesos/releases/tag/v0.5.0).
+
+        Example (v0.5.0, Mac OS X, 64-bit):
+
+        ```
+        cd bin
```
Comment (jdef) [inline docs/usage.md]: what do you think about sub-shelling to get rid of the `cd ..`?
```
(cd bin; curl -L https://... | tar -xz)
```

**Comment by jdef** [comment]
1 comment, otherwise lgtm

**Comment by karlkfi** [comment]
Untared into bin instead of cd-ing.

Squashed.

**Pull request merged**